front/src/components/pages/TwoFactorAuth/TwoFactorAuth.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles, createStyles } from '@material-ui/core/styles';
+import TextInput from '../../atoms/TextInput/TextInput';
+import Button from '../../atoms/Button/Button';
+
+const useStyles = makeStyles(() => createStyles({
+  root: {
+    width: '100%',
+    height: '100%',
+  },
+  top_bottom: {
+    padding: '50px 0',
+  },
+  mid: {
+    padding: '20px 0',
+  },
+}));
+
+const TwoFactorAuth = (): React.ReactElement => {
+  const [authCode, setAuthCode] = useState<string>('');
+  const [isFailed, setIsFailed] = useState<boolean>(false);
+  const [errorMsg, setErrorMsg] = useState<string>('');
+
+  const handleAuthCode = (event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>) => {
+    if (!(event.target instanceof HTMLInputElement)
+    && !(event.target instanceof HTMLTextAreaElement)) {
+      return;
+    }
+    setAuthCode(event.target.value);
+    if (!isFailed) {
+      return;
+    }
+    setIsFailed(false);
+  };
+  const handleTwoFactorAuth = async () => {
+    // body에 인증코드 넣기
+    console.log(authCode);
+    setIsFailed(true);
+    setErrorMsg('인증번호가 잘못되었습니다.');
+  };
+  const classes = useStyles();
+
+  return (
+    <Grid
+      className={classes.root}
+      container
+      direction="column"
+      justifyContent="center"
+      alignItems="center"
+    >
+      <Grid className={classes.top_bottom} item>
+        <h1>Mighty Pong</h1>
+      </Grid>
+      <Grid className={classes.mid} item>
+        <TextInput
+          type="text"
+          label="auth code"
```

Comment:
```suggestion
          label="Auth Code"
```
소문자로 하는것보단 대문자가 더 좋아보입니다